odo/backends/tests/test_mysql.py

```diff
@@ -202,8 +202,8 @@ def test_sql_to_csv(sql, csv):
         csv = odo(sql, fn)
         assert odo(csv, list) == data
 
-        # explicitly test that we do NOT preserve the header here
-        assert discover(csv).measure.names != discover(sql).measure.names
+        # explicitly test that we do NOT preserve the header here ???
+        #assert discover(csv).measure.name != discover(sql).measure.name
```

Comment:
Is there a reason the header should NOT be preserved? Or was it just not preserved before because we kept rediscovering the datashape?